apollo-router/src/plugins/fleet_detector.rs

```diff
@@ -291,4 +298,12 @@ fn get_otel_os() -> &'static str {
     }
 }
 
+fn get_deployment_type() -> &'static str {
+    // Official Apollo helm chart
+    if std::env::var_os(OFFICIAL_HELM_CHART_VAR).is_some() {
+        return "official_helm_chart";
```

Comment:
I guess here it does do some work: official entails apollo-made, and it's nice to have it represented in the env var to keep the language tight

Reply:
Yep! That was my thinking there. Gotta keep this forward looking 😄